index.html

```diff
@@ -7,7 +7,7 @@
 
     <!-- Bootstrap -->
     <link rel="stylesheet" href="/libs/css/bootstrap/bootstrap-3.3.7-dist/css/bootstrap.min.css" />
-   <link href="/libs/css/bootstrap/bootstrap-3.3.7-dist/flatly/bootstrap.min.css" rel="stylesheet" />-->
+   <link href="/libs/css/bootstrap/bootstrap-3.3.7-dist/flatly/bootstrap.min.css" rel="stylesheet" />
```

Comment:
We don't need both bootstrap.min.css and flatly/bootstrap.min.css. Delete the first instance.